### ISSUE TITLE
#2334 (for #2224) introduced usability issue (redundant menu)

### ISSUE THREAD

**Issue by meow81** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#2942]
**OS:**
Windows 10 (don't think OS matters here)
**Issue Descriptions:**
#2334 for #2224 via the "keep sidebar open" option created a usability issue with having two identical menus visible.  Google's web player has both of these menus, but doesn't have this issue because there's no option to keep the hamburger menu expanded or sidebar open.  

If I choose to keep the sidebar open, then the other menu should not be visible, but being that it looks like that is part of the google play web page, I don't think that is a feasible option here.  

I think that you should change this behavior back to mirror exactly as google designed it with only one of these menus visible at a time, which I suppose would be removing feature #2224.

![sidebar open redundant](https://user-images.githubusercontent.com/22875048/34705648-ee437e10-f4cf-11e7-9683-9386b83dab7b.JPG)

**Comment by MarshallOfSound** [comment]
So to be clear, you want me to remove a feature because when you turned it on (it is off by default) you didn't like the UX it gave you.  If you don't like the UX don't use the feature, I put big UI changes like this behind options so that people have the choice to use it or not.  I don't use half the option in this app but I added them so that everyone can use it however they want.

TLDR: The option stays, some people use it and like it 👍

**Comment by meow81** [comment]
Having redundant menus display isn't a good UX. Perhaps you missed the point of the feedback.  My feedback is that by implementing that option and when a user enables it they now have two display.  On Google's web app, that isn't an issue because only one is ever displayed.  So if you're going to implement an option such as that, make sure it doesn't introduce a double menu or bad design.  Do you want feedback or not?  based on the tone of your response, it doesn't appear that way.

**Comment by meow81** [comment]
forgot to mention, I want the menu displayed, so I want that feature on, but not if it's also going to display a redundant menu to the right of it.

**Comment by MarshallOfSound** [comment]
> Do you want feedback or not?

I always like feedback, but **constructive** feedback.  You're issue can effectively be boiled down to "I enabled this feature and I don't like what it did, I recommend you remove this feature" which IMO is barely feedback and definitely not constructive.  If you have a proposal to **improve** that feature in some way to meet your desired UX then raise that instead.

**Comment by meow81** [comment]
It was constructive. by creating that option, when enabled, it displays two of essentially the same menus, which you haven't acknowledged.  That's not a good UI.  my original post provided constructive feedback to say when that option is enabled, hide the redundant menu to the right of it.  what value does that provide an end user.  please at least acknowledge that i didn't just complain and did provide a recommendation.

**Comment by meow81** [comment]
Dude not trying to be argumentative because I value this app a lot. But I'm telling you from my perspective, the double display of a menu is not a great experience and it was introduced by that menu option.  I'm not telling you it's urgent or should be prioritized.  It's feedback on a feature that was added.  Can you honestly tell me you think having two of the same menus displayed is necessary or useful?
